### PR TITLE
[FIX] Translation On JSON EN

### PR DESCRIPTION
**Pull request by bernardoetrevisan**
@RocketChat/core 

I've been making some changes to the translation keys file with the goal of reducing the number o keys and simplifying the document for the translators on LingoHub. Some of the changes include deleting keys that weren't being used, simplifying the text, and correcting capitalization redundancies.

### CODE REVIEW

File: packages/rocketchat-i18n/i18n/en.i18n.json [hunk 1]
```diff
@@ -293,9 +293,7 @@
   "Avatar_url_invalid_or_error": "The url provided is invalid or not accessible. Please try again, but with a different url.",
   "away": "away",
   "Away": "Away",
-  "away_female": "away",
```
Comment (engelgabriel) [inline packages/rocketchat-i18n/i18n/en.i18n.json]: Was this key never used anywhere?

Comment (bernardoetrevisan) [reply packages/rocketchat-i18n/i18n/en.i18n.json]: Searched for it on the Rocket.Chat repository and couldn't find any location apart from the JSON files

Comment (sampaiodiego) [reply packages/rocketchat-i18n/i18n/en.i18n.json]: I don't think we have used gender for i18n so far

Comment (bernardoetrevisan) [reply packages/rocketchat-i18n/i18n/en.i18n.json]: Yes, I couldn't find it being used anywhere else but there were some keys there that differentiated gender (mostly status keys)

File: packages/rocketchat-channel-settings/client/views/channelSettings.js [hunk 2]
```diff
@@ -263,7 +263,7 @@ Template.channelSettingsEditing.onCreated(function() {
 						type: 'warning',
 						showCancelButton: true,
 						confirmButtonColor: '#DD6B55',
-						confirmButtonText: value ? t('Yes_archive_it') : t('Yes_unarchive_it'),
+						confirmButtonText: value ? t('Yes') : t('Yes'),
```
Comment (rodrigok) [inline packages/rocketchat-channel-settings/client/views/channelSettings.js]: Ok, why we have the condition here if the value will be Yes in either cases?

File: packages/rocketchat-github-enterprise/startup.js [hunk 3]
```diff
@@ -1,14 +1,14 @@
 RocketChat.settings.addGroup('OAuth', function() {
 	this.section('GitHub Enterprise', function() {
 		const enableQuery = {
-			_id: 'Accounts_OAuth_GitHub_Enterprise',
+			_id: 'Accounts_OAuth_Github',
```
Comment (rodrigok) [inline packages/rocketchat-github-enterprise/startup.js]: Do not change the setting names

File: packages/rocketchat-gitlab/startup.js [hunk 4]
```diff
@@ -1,14 +1,14 @@
 RocketChat.settings.addGroup('OAuth', function() {
 	this.section('GitLab', function() {
 		const enableQuery = {
-			_id: 'Accounts_OAuth_Gitlab',
+			_id: 'Accounts_OAuth_Github',
```
Comment (rodrigok) [inline packages/rocketchat-gitlab/startup.js]: Do not change the settings name

File: packages/rocketchat-ldap/server/ldap.js [hunk 5]
```diff
@@ -17,8 +17,8 @@ export default class LDAP {
 		this.connected = false;
 
 		this.options = {
-			host: RocketChat.settings.get('LDAP_Host'),
-			port: RocketChat.settings.get('LDAP_Port'),
+			host: RocketChat.settings.get('Host'),
```
Comment (rodrigok) [inline packages/rocketchat-ldap/server/ldap.js]: Do not change the setting name

File: packages/rocketchat-ldap/server/ldap.js [hunk 6]
```diff
@@ -17,8 +17,8 @@ export default class LDAP {
...
+			port: RocketChat.settings.get('Port'),
```
Comment (rodrigok) [inline packages/rocketchat-ldap/server/ldap.js]: Do not change the setting name

File: packages/rocketchat-ldap/server/ldap.js [hunk 7]
```diff
@@ -27,12 +27,12 @@ export default class LDAP {
 			encryption: RocketChat.settings.get('LDAP_Encryption'),
 			ca_cert: RocketChat.settings.get('LDAP_CA_Cert'),
 			reject_unauthorized: RocketChat.settings.get('LDAP_Reject_Unauthorized') || false,
-			Authentication: RocketChat.settings.get('LDAP_Authentication'),
+			Authentication: RocketChat.settings.get('Enable'),
```
Comment (rodrigok) [inline packages/rocketchat-ldap/server/ldap.js]: Do not change the setting name

File: packages/rocketchat-ldap/server/ldap.js [hunk 8]
```diff
@@ -27,12 +27,12 @@ export default class LDAP {
...
 			Authentication_UserDN: RocketChat.settings.get('LDAP_Authentication_UserDN'),
-			Authentication_Password: RocketChat.settings.get('LDAP_Authentication_Password'),
+			Authentication_Password: RocketChat.settings.get('Password'),
```
Comment (rodrigok) [inline packages/rocketchat-ldap/server/ldap.js]: Do not change the setting name

File: packages/rocketchat-ldap/server/ldap.js [hunk 9]
```diff
@@ -27,12 +27,12 @@ export default class LDAP {
...
 			BaseDN: RocketChat.settings.get('LDAP_BaseDN'),
 			User_Search_Filter: RocketChat.settings.get('LDAP_User_Search_Filter'),
-			User_Search_Scope: RocketChat.settings.get('LDAP_User_Search_Scope'),
+			User_Search_Scope: RocketChat.settings.get('Scope'),
```
Comment (rodrigok) [inline packages/rocketchat-ldap/server/ldap.js]: Do not change the setting name

File: packages/rocketchat-ldap/server/loginHandler.js [hunk 10]
```diff
@@ -34,7 +34,7 @@ Accounts.registerLoginHandler('ldap', function(loginRequest) {
 
 	logger.info('Init LDAP login', loginRequest.username);
 
-	if (RocketChat.settings.get('LDAP_Enable') !== true) {
+	if (RocketChat.settings.get('Enable') !== true) {
```
Comment (rodrigok) [inline packages/rocketchat-ldap/server/loginHandler.js]: Do not change the setting name

File: packages/rocketchat-ldap/server/loginHandler.js [hunk 11]
```diff
@@ -91,7 +91,7 @@ Accounts.registerLoginHandler('ldap', function(loginRequest) {
 
 	let username;
 
-	if (RocketChat.settings.get('LDAP_Username_Field') !== '') {
+	if (RocketChat.settings.get('Accounts_OAuth_Custom_Username_Field') !== '') {
```
Comment (rodrigok) [inline packages/rocketchat-ldap/server/loginHandler.js]: Do not change the setting name

File: packages/rocketchat-ldap/server/loginHandler.js [hunk 12]
```diff
@@ -138,7 +138,7 @@ Accounts.registerLoginHandler('ldap', function(loginRequest) {
 
 	logger.info('User does not exist, creating', username);
 
-	if (RocketChat.settings.get('LDAP_Username_Field') === '') {
+	if (RocketChat.settings.get('Accounts_OAuth_Custom_Username_Field') === '') {
```
Comment (rodrigok) [inline packages/rocketchat-ldap/server/loginHandler.js]: Do not change the setting name

File: packages/rocketchat-ldap/server/settings.js [hunk 13]
```diff
@@ -1,8 +1,8 @@
 RocketChat.settings.addGroup('LDAP', function() {
-	const enableQuery = {_id: 'LDAP_Enable', value: true};
+	const enableQuery = {_id: 'Enable', value: true};
```
Comment (rodrigok) [inline packages/rocketchat-ldap/server/settings.js]: Do not change the setting names

File: packages/rocketchat-ldap/server/sync.js [hunk 14]
```diff
@@ -24,7 +24,7 @@ export function getPropertyValue(obj, key) {
 
 
 export function getLdapUsername(ldapUser) {
-	const usernameField = RocketChat.settings.get('LDAP_Username_Field');
+	const usernameField = RocketChat.settings.get('Accounts_OAuth_Custom_Username_Field');
```
Comment (rodrigok) [inline packages/rocketchat-ldap/server/sync.js]: Do not change the setting names

File: packages/rocketchat-ldap/server/syncUsers.js [hunk 15]
```diff
@@ -11,7 +11,7 @@ Meteor.methods({
 			throw new Meteor.Error('error-not-authorized', 'Not authorized', { method: 'ldap_sync_users' });
 		}
 
-		if (RocketChat.settings.get('LDAP_Enable') !== true) {
+		if (RocketChat.settings.get('Enable') !== true) {
```
Comment (rodrigok) [inline packages/rocketchat-ldap/server/syncUsers.js]: Do not change the setting name

File: packages/rocketchat-ldap/server/testConnection.js [hunk 16]
```diff
@@ -11,7 +11,7 @@ Meteor.methods({
 			throw new Meteor.Error('error-not-authorized', 'Not authorized', { method: 'ldap_test_connection' });
 		}
 
-		if (RocketChat.settings.get('LDAP_Enable') !== true) {
+		if (RocketChat.settings.get('Enable') !== true) {
```
Comment (rodrigok) [inline packages/rocketchat-ldap/server/testConnection.js]: Do not change the setting name

File: packages/rocketchat-lib/server/functions/saveUser.js [hunk 17]
```diff
@@ -11,15 +11,15 @@ RocketChat.saveUser = function(userId, userData) {
 	}
 
 	if (!userData._id && !RocketChat.authz.hasPermission(userId, 'create-user')) {
-		throw new Meteor.Error('error-action-not-allowed', 'Adding user is not allowed', { method: 'insertOrUpdateUser', action: 'Adding_user' });
+		throw new Meteor.Error('error-action-not-allowed', 'Adding user is not allowed', { method: 'insertOrUpdateUser', action: 'Loading...' });
```
Comment (rodrigok) [inline packages/rocketchat-lib/server/functions/saveUser.js]: This describe the action, please do not change

File: packages/rocketchat-livechat/client/route.js [hunk 18]
```diff
@@ -23,7 +23,7 @@ AccountBox.addRoute({
 	name: 'livechat-users',
 	path: '/users',
 	sideNav: 'livechatFlex',
-	i18nPageTitle: 'Livechat_Users',
+	i18nPageTitle: 'Users',
```
Comment (rodrigok) [inline packages/rocketchat-livechat/client/route.js]: These describe the page you are in, I don't think it's good to change them

File: packages/rocketchat-i18n/i18n/en.i18n.json [hunk 19]
```diff
@@ -845,7 +813,7 @@
   "How_friendly_was_the_chat_agent": "How friendly was the chat agent?",
   "How_knowledgeable_was_the_chat_agent": "How knowledgeable was the chat agent?",
   "How_long_to_wait_after_agent_goes_offline": "How Long to Wait After Agent Goes Offline",
-  "How_responsive_was_the_chat_agent": "How responsive was the chat agent?",
+  "How_responsive_was_the_chat_agent": "How responsive was the chat agent?",
```
Comment (MartinSchoeler) [inline packages/rocketchat-i18n/i18n/en.i18n.json]: There should be no trailing spaces on this line.

File: packages/rocketchat-i18n/i18n/en.i18n.json [hunk 20]
```diff
@@ -1058,26 +1015,22 @@
   "LDAP_CA_Cert": "CA Cert",
   "LDAP_Connect_Timeout": "Connection Timeout (ms)",
   "LDAP_Default_Domain": "Default Domain",
-  "LDAP_Default_Domain_Description": "If provided the Default Domain will be used to create an unique email for users where email was not imported from LDAP. The email will be mounted as `username@default_domain` or `unique_id@default_domain`.<br/>Example: `rocket.chat`",
+  "LDAP_Default_Domain_Description": "If provided the Default Domain will be used to create an unique email for users where email was not imported from LDAP. The email will be mounted as `username@default_domain` or `unique_id@default_domain`.<br/>Example: `rocket.chat`",
```
Comment (MartinSchoeler) [inline packages/rocketchat-i18n/i18n/en.i18n.json]: More trailing spaces

**Pull request closed without merging**